dashboard/app/views/plc/enrollment_unit_assignments/_form.html.haml

```diff
@@ -0,0 +1,19 @@
+= form_for @plc_enrollment_unit_assignment do |f|
```

Comment:
Basic scaffolding for the unit assignment - I didn't change this beyond what was autogenerated and it will get stripped out soonish